demisto_sdk/commands/download/downloader.py

```diff
@@ -216,7 +225,7 @@ def build_pack_content(self) -> None:
                 # If entity is of type integration/script it will have dirs, otherwise files
                 entity_instances_paths: list = get_child_directories(content_entity_path)
             else:
-                entity_instances_paths: list = get_child_files(content_entity_path)
```

Comment:
why did you change this? whats returned from the func?

Reply:
List is returned and we should annotate

Reply:
we should not double annotate.
![image](https://user-images.githubusercontent.com/11165655/80954754-a9ec3900-8e06-11ea-923b-40326288e28a.png)